### DISCUSSION TITLE
Close the inline suggestion popup

### DISCUSSION THREAD

**Discussion by black-pwq** [VSCodeVim/Vim#9218] [Q&A] [answered]
Is there a way to close the inline suggestion window when vim mode is enabled? "Escape" key is exclusively used in vim which, however, is the default key to close inline suggestion in vscode. I know a quick fix is to remap it to other key. However, as an unexperienced vim user, I am curious about how inline suggestion is closed in vim itself. After googling and practice, I found ctrl-e is the one works (at least in command line), but not work in vscode-vim (still duplicates the character under the cursor). I know there are extensions in vim that bring up the inline suggestion, then how do they close it? And does that work in vscode-vim?

**Comment by saschafuchs** [comment, marked as answer]
Use shift esc to close the suggestion window.
